### DISCUSSION TITLE
Any idea why boiler flow and return temp seem stuck to 25?

### DISCUSSION THREAD

**Discussion by JRascagneres** [gekkekoe/esphome-ecodan-hp#94] [Announcements]
![Screenshot_20241220-205909](https://github.com/user-attachments/assets/c81c33a6-3946-4432-8cf9-7fa572214264)

**Comment by gekkekoe** [comment]
because you probably don't have the sensors fitted...

**Reply by JRascagneres** [reply]
Ah are they additional sensors?

**Reply by gekkekoe** [reply]
yup
